.github/actions/prepare/action.yml

```diff
@@ -23,3 +23,8 @@ runs:
       run: |
         python manage.py makemigrations
         python manage.py migrate
+    - name: spin up redis with docker
+      shell: bash
+      working-directory: voluncheer
+      run: |
+        docker run -p 6379:6379 -d redis:5
```

Comment:
GitHub actions has a service container for Redis, we should use that instead: https://docs.github.com/en/actions/using-containerized-services/creating-redis-service-containers

Reply:
Right now it doesn't interfere with CI. Maybe I can work on this next week? I haven't built the test suite yet due to time-constraint.

Reply:
We aren't rushing this in so let's get this right.

Reply:
Do you think I would need to test for chatroom in CI though? If so, what kind of test should that be? I am already testing for chatroom model and view access, and those doesn't seem to require redis.

Reply:
If the testing works without a redis provider we shouldn't need this at all.

Reply:
Please remove if we are indeed not using this.